keps/prod-readiness/sig-scheduling/4832.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4832
+alpha:
+  approver: "@wojtek-t"
```

Comment:
cc @wojtek-t 

🙏

Reply:
I will take a look after preliminary buy-in from the SIG